release-notes.md

```diff
@@ -1,3 +1,7 @@
+## 1 February 2017
+### New features
+* We're working on some fancy new things in dev so our only new feature released this week is retrying timed-out elasticsearch requests.  By default, Pelias will, by default, retry a timed-out request up to 3 times before failing for good, though this number can be overridden by specifying `api.requestRetries` in your Pelias configuration.  
```

Comment:
I wouldn't mention this in mapzen search docs. Just say that the release is mostly a data refresh.